### PR TITLE
Fix spellings

### PR DESCRIPTION
**Pull request by loic-sharma**

### CODE REVIEW

File: docs/consume-packages/Package-References-in-Project-Files.md [hunk 1]
```diff
@@ -189,9 +189,9 @@ If this property is set, NuGet restore will generate a lock file - `packages.loc
 ### `restore` behavior with lock file
 If a lock file is present for project, NuGet uses this lock file to run `restore`. NuGet does a quick check to see if there were any changes in the package dependencies as mentioned in the project file (or dependent projects' files) and if there were no changes it just restores the packages mentioned in the lock file. There is no re-evaluation of package dependencies.
 
-If NuGet detects a change in the defined dependenices as mentioned in the project file(s), it re-evaluates the package graph and updates the lock file to reflect the new package closure for the project.
+If NuGet detects a change in the defined dependencies as mentioned in the project file(s), it re-evaluates the package graph and updates the lock file to reflect the new package closure for the project.
 
-For CI/CD and other scenarios, where you would not want to change the package dependenies on the fly, you can do so by setting the `lockedmode` to `true`:
+For CI/CD and other scenarios, where you would not want to change the package dependencies on the fly, you can do so by setting the `lockedmode` to `true`:
```
Comment (loic-sharma) [inline docs/consume-packages/Package-References-in-Project-Files.md]: Should `lockedmode` be `locked-mode`?  The example command below is: `dotnet.exe restore --locked-mode`

Comment (loic-sharma) [reply docs/consume-packages/Package-References-in-Project-Files.md]: @jainaashish

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-3/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1172) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811192217133567-1172%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ipXaywdJt7P0wgNf6kSCwvgpVMX2zVTZBYtHbuyG22c%3D&st=2018-11-19T22%3A13%3A12Z&se=2018-12-20T22%3A18%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/consume-packages/Package-References-in-Project-Files.md (hunk 1 above)
Comment (nkolev92) [reply docs/consume-packages/Package-References-in-Project-Files.md]: Is `lockedmode` something that can be set anywhere? 

Can we reword it to say "enabling locked mode"? 
Because you need to do a different thing in dotnet.exe/ if running throguh msbuild.

**Comment by zivkan** [comment]
I created a new issue for the comments about changing the locked mode wording, so I can get these spelling fixes merged.
